core/src/test/scala/org/locationtech/rasterframes/MaskingFunctionsSpec.scala

```diff
@@ -0,0 +1,439 @@
+/*
+ * This software is licensed under the Apache 2 license, quoted below.
+ *
+ * Copyright 2019 Astraea, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     [http://www.apache.org/licenses/LICENSE-2.0]
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.locationtech.rasterframes
+
+import geotrellis.raster
+import geotrellis.raster._
+import geotrellis.raster.testkit.RasterMatchers
+import org.apache.spark.sql.Encoders
+import org.apache.spark.sql.functions._
+import org.locationtech.rasterframes.expressions.accessors.ExtractTile
+import org.locationtech.rasterframes.stats._
+import org.locationtech.rasterframes.tiles.ProjectedRasterTile
+
+
+import geotrellis.raster.testkit.RasterMatchers
+
+class MaskingFunctionsSpec extends TestEnvironment with RasterMatchers {
```

Comment:
Note we have refactored the mask related function tests out into this spec. Just because we had RasterFunctionsSpec gettting so big and starting to get disorganized.

Reply:
Put this in the `functions` package. I've continued down this vein in https://github.com/locationtech/rasterframes/pull/430